docs/commands/bundle.md

```diff
@@ -42,6 +42,7 @@ Option                 | Type      | Required?    | Default     | Description
 `--skip-preprocessor`  | `array`   | no           | -           | Ignore certain preprocessors. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator) below
 `--skip-rule`          | `array`   | no           | -           | Ignore certain rules. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator) below
 `--version`            | `boolean` | no           | -           | Show version number
+`--clean-components`   | `boolean` | no           | -           | Clean unused components
```

Comment:
This should be `remove-unused-components`. Clean can mean different things to different people.

Reply:
@adamaltman done.